plugins/modules/system/sap_task_list_execute.py

```diff
@@ -0,0 +1,416 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2021, Rainer Leber <rainerleber@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: sap_task_list_execute
+short_description: Perform SAP Task list execution.
+version_added: "3.5.0"
+description:
+  - The C(sap_task_list_execute) module will perform RFC execution on SAP Systems
+    for tasks lists.
+  - Tasks in the task list which requires manual activities will be confirmed automatically.
+  - This module will use the RFC Package C(STC_TM_API).
```

Comment:
@rainerleber I saw this now, and I thought of the `requirements` item in the documentation blob. I think you should add it, tomark the dependency on `pyrfc`, and as that happens, this line becomes redundant.

Reply:
@rainerleber ou marked as resolved, but I still don't see the `requirements` key in here. As an example:
https://github.com/ansible-collections/community.general/blob/main/plugins/modules/system/pids.py#L14-L15

Reply:
adding another comment to become part of a review. The real stuff is at ^